tests/tools/onert_train/test/rawdataloader.test.cc

```diff
@@ -124,16 +130,25 @@ TEST_F(RawDataLoaderTest, loadDatas_1)
   std::vector<nnfw_tensorinfo> expected_infos{expected_info};
 
   // Generate test data
-  std::vector<uint32_t> in(num_input * num_elems(&in_info));
-  std::generate(in.begin(), in.end(), [] {
-    static uint32_t i = 0;
-    return i++;
-  });
-  std::vector<uint32_t> expected(num_expected * num_elems(&expected_info));
-  std::generate(expected.begin(), expected.end(), [in] {
-    auto sum = std::accumulate(in.begin(), in.end(), 0);
-    return sum;
-  });
+  std::vector<std::vector<uint32_t>> in(num_input);
+  for (uint32_t i = 0; i < num_input; ++i)
+  {
+    in[i].resize(num_elems(&in_infos[i]));
+    std::generate(in[i].begin(), in[i].end(), [this] {
+      static uint32_t i = 0;
+      return i++;
+    });
+  }
+
+  std::vector<std::vector<uint32_t>> expected(num_expected);
+  for (uint32_t i = 0; i < num_expected; ++i)
+  {
+    expected[i].resize(num_elems(&expected_infos[i]));
+    std::generate(expected[i].begin(), expected[i].end(), [in, i] {
+      auto sum = std::accumulate(in[i].begin(), in[i].end(), 0);
+      return sum;
```

Comment:
@jyoungyun What model do you use to compare this expected values?

Reply:
I haven't compared the expected value. I made expected value using the dump output data from `onert_run`. I used mnist model and more simple model for easy test.

Reply:
@jyoungyun If you use the dump of `onert_run`, why `std::accumulate` and `std::genereate` are there? I guessed you're manipulating and calculate something.

Reply:
Ah, I misunderstood. This PR is for unittest case, so I did not consider the model. It generates arbitrarily input and expected data. Should I consider a model for unittest?

Reply:
I understood. The expected value can be replaced with random generated number. This PR is for testing input and expected is loaded correctly.